sqlglot/dialects/dameng.py

```diff
@@ -5,18 +5,11 @@
 
 class Dameng(Oracle):
     class Parser(Oracle.Parser):
-        FUNCTIONS = {
-            **Oracle.Parser.FUNCTIONS
-        }
+        FUNCTIONS = {**Oracle.Parser.FUNCTIONS}
```

Comment:
none of these are necessary, please delete

Reply:
> none of these are necessary, please delete

Done. Thank you for your review.